.travis.yml

```diff
@@ -32,6 +32,7 @@ addons:
       - g++-6
   firefox: latest
 before_script:
+  - export CHROME_BIN=chromium-browser
```

Comment:
I don't thing it will work because of sandboxing issues, but let's see what Travis says. Maybe it will work this time.